core-bundle/src/Resources/contao/templates/picture/picture_default.html5

```diff
@@ -1,12 +1,18 @@
+<?php $hasSingleAspectRatio = true; ?>
 
 <?php if ($this->sources): ?>
   <picture>
     <?php foreach ($this->sources as $source): ?>
       <source srcset="<?= $source['srcset'] ?>"<?php if (!empty($source['sizes'])): ?> sizes="<?= $source['sizes'] ?>"<?php endif; ?><?php if (!empty($source['media'])): ?> media="<?= $source['media'] ?>"<?php endif; ?><?php if (!empty($source['type'])): ?> type="<?= $source['type'] ?>"<?php endif; ?>>
+      <?php
+        if (!empty($this->img['width']) && !empty($this->img['height']) && !empty($source['width']) && !empty($source['height']) && abs(($this->img['width'] / $this->img['height']) / ($source['width'] / $source['height']) - 1) > 0.05) {
```

Comment:
```suggestion
        if (true === $hasSingleAspectRatio && !empty($this->img['width']) && !empty($this->img['height']) && !empty($source['width']) && !empty($source['height']) && abs(($this->img['width'] / $this->img['height']) / ($source['width'] / $source['height']) - 1) > 0.05) {
```

This way it will skip the checks if `$hasSingleAspectRation` is already `false`.